connect/service.go

```diff
@@ -5,12 +5,13 @@ import (
 	"github.com/micromdm/micromdm/command"
 	"github.com/micromdm/micromdm/device"
 	"github.com/pkg/errors"
+	"golang.org/x/net/context"
 )
 
 // Service defines methods for an MDM service
 type Service interface {
-	Acknowledge(deviceUDID, commandUUID string) (int, error)
-	NextCommand(deviceUDID string) ([]byte, int, error)
+	Acknowledge(ctx context.Context, req mdmConnectRequest) (int, error)
```

Comment:
The `req` argument should be the exported mdm.Response instead of the command request. Having an un-exported type here makes it hard to call Service outside the package.